projects/media-viewer/src/lib/annotations/annotation.component.html

```diff
@@ -1,21 +1,22 @@
-<div class="annotation-wrapper" tabindex="1" (focusout)="toggleSelection(false)">
+<ng-container *ngFor="let rectangle of annotation.rectangles; index as i">
   <mv-anno-rectangle
     (click)="toggleSelection(true)"
-    (update)="onRectangleUpdate($event)"
     [selected]="selected"
     [zoom]="zoom"
-    [rectangle]="annotation.rectangles[0]"
+    [rectangle]="rectangle"
     [draggable]="draggable"
     [color]="annotation.color">
   </mv-anno-rectangle>
-  <mv-anno-comment
-    *ngIf="annotation.comments[0]"
-    (click)="toggleSelection(true)"
-    (delete)="deleteComment()"
-    (updated)="onCommentUpdate($event)"
-    [selected]="selected"
-    [comment]="annotation.comments[0]"
-    [left]="commentsLeftOffset"
-    [top]="annotation.rectangles[0].y *(zoom)">
-  </mv-anno-comment>
-</div>
+</ng-container>
+
+<mv-anno-comment
+  *ngIf="annotation.comments[0]"
```

Comment:
do we need a for loop here to go through the comments?

Reply:
We're replicating the JUI behaviour - only one comment per highlight